explorer/src/components/account/AnchorAccountCard.tsx

```diff
@@ -0,0 +1,175 @@
+        }
+      />
+    );
+  }
+
+  return (
+    <>
+      <div className="card">
+        <div className="card-header">
+          <div className="row align-items-center">
+            <div className="col">
+              <h3 className="card-header-title">{foundAccountLayoutName}</h3>
+            </div>
+          </div>
+        </div>
+
+        <div className="table-responsive mb-0">
+          <table className="table table-sm table-nowrap card-table">
+            <thead>
+              <tr>
+                <th className="w-1 text-muted">Key</th>
+                <th className="text-muted">Value</th>
+              </tr>
+            </thead>
+            <tbody className="list">
+              {decodedAnchorAccountData &&
+                Object.keys(decodedAnchorAccountData).map((key) => (
+                  <AccountRow
+                    key={key}
+                    valueName={key}
+                    // @ts-ignore
+                    value={decodedAnchorAccountData[key]}
+                  />
+                ))}
+            </tbody>
+          </table>
+        </div>
+        <div className="card-footer">
+          <div className="text-muted text-center">
+            {decodedAnchorAccountData &&
+            Object.keys(decodedAnchorAccountData).length > 0
+              ? `Decoded ${Object.keys(decodedAnchorAccountData).length} Items`
+              : "No decoded data"}
+          </div>
+        </div>
+      </div>
+    </>
+  );
+}
+
+function AccountRow({ valueName, value }: { valueName: string; value: any }) {
+  let displayValue: JSX.Element;
+  if (value && value.constructor && value.constructor.name === "PublicKey") {
+    displayValue = <Address pubkey={value} link />;
+  } else if (value && value.constructor && value.constructor.name === "BN") {
+    displayValue = <>{value.toString()}</>;
+  } else if (value && typeof value !== "object") {
+    displayValue = <>{String(value)}</>;
+  } else if (value) {
+    const displayObject = createDisplayObject(value);
+    displayValue = (
+      <ReactJson
+        src={JSON.parse(JSON.stringify(displayObject))}
+        collapsed={1}
+        theme="solarized"
+      />
+    );
+  } else {
+    displayValue = <>null</>;
+  }
+  return (
+    <tr>
+      <td className="w-1 text-monospace">{camelToUnderscore(valueName)}</td>
+      <td className="text-monospace">{displayValue}</td>
+    </tr>
+  );
+}
+
+function camelToUnderscore(key: string) {
+  var result = key.replace(/([A-Z])/g, " $1");
+  return result.split(" ").join("_").toLowerCase();
+}
+
+function createDisplayObject(object: Object): Object {
+  if (!Array.isArray(object)) {
+    if (
+      object &&
+      object.constructor &&
+      object.constructor.name === "PublicKey"
+    ) {
+      return object.toString();
+    } else if (
+      object &&
+      object.constructor &&
+      object.constructor.name === "BN"
+    ) {
+      // @ts-ignore
+      return object.toNumber();
```

Comment:
This sounds like a bad idea, since JS numbers are not 64-bit. Can you keep it a string all the way through to display?